app/components/sideNavOne.tsx

```diff
@@ -55,7 +55,7 @@ const bottomNav = [
 ];
 const SideNavigation = () => {
   return (
-    <div className="hidden lg:block col-span-3 h-full bg-white overflow-y-scroll relative">
+    <div className="hidden lg:block min-w-[240px] min-h-screen bg-white overflow-y-scroll relative">
       <div className="pt-[58px] pl-[38px] pr-[15px]">
         <h4 className="text-3xl xl:text-4xl font-semibold text-black">
```

Comment:
Not an exact replica of the design file
<img width="1437" alt="Screenshot 2024-12-20 at 10 51 44" src="https://github.com/user-attachments/assets/7028b002-13c9-4d4e-8d8a-2e870acf2274" />

Reply:
also the side nav doesnt hightlight the active link as show in the design file

Reply:
I was quite hesitant to edit the Side nav and the event card components as I thought I only had to reuse them and not change them. I'll make the necessary changes now.

Reply:
The side nav now highlights the active link as show in the design file